src/OrchardCore.Modules/OrchardCore.OpenId/Configuration/OpenIdValidationConfiguration.cs

```diff
@@ -77,30 +75,13 @@ public void Configure(AuthenticationOptions options)
                     return;
                 }
 
-                // Register the JWT or validation handler in the authentication handlers collection.
```

Comment:
In 3.0, the OpenIddict validation handler will support both JWT and Data Protection tokens.